File: flutter-main/flutter-main/Inheritence/p5.dart
class Parent{
	int x = 10;
	Parent(){
		print("Parent Constructor");
		print(this.hashCode);
		}
		void printData(){
			print(x);
		}
	
}
class Child extends Parent{
	int x = 20;
	Child(){
		print("Child Constructor");
		print(this.hashCode);
	}
	void dispData(){
	print(x);
	}
}
void main(){
	Child obj = new Child();	
	obj.dispData();
	obj.printData();
}

File: java-main/java-main/Inheritence/p5.java
//static in parentclass...


class Parent{
	static{
		System.out.println("In Parent static block");
	}
}
class child extends Parent{
	static{
		System.out.println("IN child static block");
	}
}
class client{
	public static void main(String[]args){
		child obj=new child();
	}
}

